src/lib.ts

```diff
@@ -0,0 +1,37 @@
+import { getVersion, getLastPart } from "./utils";
+
+/**
+ * ******************************************************
+ * LIB
+ * ------------------------------------------------------
+ * @description functions specific to the project
+ * @summary functions specific to the project that rely on utils
+ *
+ * ******************************************************
+ */
+
+/**
+ * getPackageNameVersionFromUrl
+ * @description a convenience function to get the package name and version from a url
+ * @param {string} url
+ * @param {boolean} debug
+ * @returns {string}
+ */
+export const getPackageNameVersionFromUrl = (url: string, debug = false) => {
```

Comment:
We do have a function for this that can be pulled from @jspm/generator here - https://jspm.org/docs/generator/stable/functions/parseUrlPkg.html. It will respect any custom CDN configurations etc and different versioning URL schemes.